src/index.html

```diff
@@ -4,9 +4,353 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="content">
+    <header class="header content__header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <main class="catalog content__catalog">
+      <article class="card catalog__card" data-qa="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="27 inch Apple imac with 5k Retina display"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          class="card__buy-button"
+          href="#buy"
+          data-qa="card-hover"
+        >Buy
```

Comment:
just a little recommendation, I think it's better to write a text from the new line
```suggestion
        >
          Buy
```

Reply:
Agree, format your code properly